.gitignore

```diff
@@ -5,4 +5,13 @@
 __pycache__
 tmp
 *.log
+*.xml
 test_results.txt
+artifacts
+cprofile
+*.prof
+
+# Test exclusions
+qa/L0_openai/openai
+tensorrtllm_models
+custom_tokenizer
```

Comment:
These are primarily for local dev/debug loop convenience, and to avoid accidentally committing them if keeping them around during dev. 

For example:
- `custom_tokenizer`: not re-downloading a pre-downloaded tokenizer (the test is focused on testing tokenizer from local file path, not on downloading a tokenizer, but downloads if not present) 
- `tensorrtllm_models`: not re-generating trtllm models each time you want to re-run individual test cases locally after the first run